src/en/controllers-gui.md

```diff
@@ -27,6 +27,14 @@ Your login credential is:
   password: 1d191f0ef257a3fc3af6be0814f6f1b0
 ```
 
+From Juju 2.0 onwards, the GUI *should* be installed by default.
+However if you see the error `ERROR Juju GUI is not available: Juju GUI not found`, then:
```

Comment:
I'd remove line 30 entirely and modify this one to read...

"If the `juju gui` command returns the error `ERROR Juju GUI is not available: Juju GUI not found`, then:"

Reply:
Ok, I just updated that